File: nodejs/ibm-multi-demo/employee.js
class Employee{
    employeeId;
    firstName;
    salary;
   
    constructor(employeeId,firstName,salary){
            this.employeeId=employeeId;
            this.firstName=firstName;
            this.salary=salary;
    };

    printEmpData=()=>{
        return `Employee: employeeId: ${this.employeeId} firstName:${this.firstName} salary:${this.salary} `;
    }

    
};
// Employee employee=new Employee();  -java
//in javascript:

const employee1=new Employee(101,"diya",10.5);
console.log(employee1.printEmpData());
const employee2=new Employee();
console.log(employee2.printEmpData());

File: workspace-spring-tool-suite-4-4.22.0.RELEASE/javaDemo/src/oop/Employee.java
package oop;

public class Employee {

	int employeeId;
	String firstName;
	double salary;

	String getEmployeeData() {
		return "Employee id: " + employeeId + " Name: " + firstName + " salary: " + salary;
	}
	public Employee(int employeeId,String firstName,double salary) {
		super();  
		this.employeeId=employeeId;
		this.firstName=firstName;
		this.salary=salary;
	}
	public Employee() {
		System.out.println("Default constructor");
	}
	
	public int getEmployeeId() {
		return employeeId;
	}
	public void setEmployeeId(int employeeId) {
		this.employeeId=employeeId;
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName=firstName;
	}
	public double getSalary() {
		return salary;
	}
	public void setSalary(double salary) {
		this.salary=salary;
	}

}